src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
+    private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("dd.MM.yyyy");
+    private static final String REPORT_HEADER = "Report for period ";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder report = new StringBuilder();
+        try {
+            Date fromDate = DATE_FORMAT.parse(dateFrom);
+            Date toDate = DATE_FORMAT.parse(dateTo);
```

Comment:
Checklist item
https://github.com/mate-academy/jv-salary-info/blob/master/checklist.md#use-localdate-instead-of-date

Reply:
not fixed